### PR TITLE
Added att_scn

### PR DESCRIPTION
**Pull request by a-kleiman**

### CODE REVIEW

File: Attribute Analysis.ipynb [hunk 1]
```diff
@@ -971,6 +974,122 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stats_info = pickle.load(open('results/{}/att_scn.pkl'.format(folder_name), 'rb'))\n",
+    "scenes_per = stats_info['scenes_per']\n",
+    "print(len(scenes_per[0]))\n",
```
Comment (Angelina-Wang) [inline Attribute Analysis.ipynb]: please remove

File: Attribute Analysis.ipynb [hunk 2]
```diff
@@ -971,6 +974,122 @@
...
+    "info = pickle.load(open('util_files/places_scene_info.pkl', 'rb'))\n",
+    "idx_to_scene = info['idx_to_scene']\n",
+    "idx_to_scenegroup = info['idx_to_scenegroup']\n",
+    "sceneidx_to_scenegroupidx = info['sceneidx_to_scenegroupidx']\n",
+    "\n",
+    "xaxis = [idx_to_scenegroup[i] for i in range(len(idx_to_scenegroup))]\n",
+    "xaxis = ['\\n'.join(textwrap.wrap(chunk, width=30)) for chunk in xaxis]\n",
+    "barWidth = .6\n",
```
Comment (Angelina-Wang) [inline Attribute Analysis.ipynb:1369]: can this be set to a constant or is it dependent on how many attributes there are?

File: Attribute Analysis.ipynb [hunk 3]
```diff
@@ -971,6 +974,122 @@
...
+    "for a in range(num_attrs):\n",
+    "    scenes_attribute[a] = np.array([scenes_per[i][a] for i in range(len(r1))]) / dataset.num_skin_tone_images[a]\n",
+    "    \n",
+    "def show_scenes():\n",
+    "    fig = plt.figure(figsize=(10, 5))\n",
+    "    \n",
+    "    scene_attr_ratios = [0 for i in range(len(scenes_attribute[0]))]\n",
+    "    for scene in range(len(scenes_attribute[0])):\n",
+    "        max_scene = 0\n",
+    "        min_scene = 1\n",
```
Comment (Angelina-Wang) [inline Attribute Analysis.ipynb:1034]: pretty sure you can't set min_scene to 1 here, it should be like float('inf') or something?

File: Attribute Analysis.ipynb [hunk 4]
```diff
@@ -44,7 +44,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (Angelina-Wang) [inline Attribute Analysis.ipynb]: please clear all cells and outputs before pushing

File: Attribute Analysis.ipynb [hunk 5]
```diff
@@ -971,6 +1009,144 @@
...
+    "    plt.yticks(ticks, np.array(xaxis)[order], fontsize=fontsize)\n",
+    "    plt.xticks(fontsize=fontsize)\n",
+    "    plt.ylabel('Scene', fontsize=fontsize, labelpad=20)\n",
+    "    plt.xlabel('Fraction of Images with this Scene', fontsize=fontsize, labelpad=20, x=.3)\n",
+    "    plt.legend(loc='best', fontsize=fontsize)\n",
+    "    plt.tight_layout()\n",
+    "    plt.gcf().subplots_adjust(bottom=0.15)\n",
+    "    plt.gcf().subplots_adjust(left=0.5)\n",
+    "    if first_pass:\n",
+    "        to_write[7] = [\"(att_scn) Distribution of scenes that each gender appears in, sorted by ratio between the two.\"]\n",
```
Comment (ryanzhang22) [inline Attribute Analysis.ipynb]: I think the references to gender should to updated to attribute

**Pull request merged**